src/FsLibLog/FsLibLog.fs

```diff
@@ -1211,4 +1196,21 @@ module LogProvider =
     let getCurrentLogger () =
         let stackFrame = StackFrame(2, false)
         getLoggerByType (stackFrame.GetMethod().DeclaringType)
+
+
+type LogProvider =
+    /// <summary>
+    /// Creates a logger based on `Reflection.MethodBase.GetCurrentMethod().FullName` and `CallerMemberName`.  This is only useful for calls within functions. This does work on inlined functions.
+    /// </summary>
+    /// <param name="memberName"></param>
+    /// <returns></returns>
+    static member inline getLoggerByFunc([<System.Runtime.CompilerServices.CallerMemberName>] ?memberName: string) =
```

Comment:
`CallerMemberName` is only available for member functions so I had to remove the FSharpFunc version.  This change _should_ be transparent to callers.